Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -76,6 +76,7 @@ open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: ChatSwipeableList
             trailing: avatarView.directionalLayoutMargins.trailing
         )
         
+        avatarView.heightAnchor.pin(equalToConstant: 45).isActive = true
```

Comment:
`48` ? 😱

Reply:
Fix was on the way when u were writing this comment :) 
I was adding a couple of more improvements from the design review.